src/wazuh_modules/vulnerability_scanner/tests/databaseFeedManager/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+cmake_minimum_required(VERSION 3.12.4)
+
+project(database_feed_manager_tests)
+
+include_directories(${SRC_FOLDER}/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/)
+include_directories(${SRC_FOLDER}/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/)
+include_directories(${SRC_FOLDER}/shared_modules/)
+include_directories(${SRC_FOLDER}/shared_modules/content_manager/src/)
+include_directories(${SRC_FOLDER}/wazuh_modules/vulnerability_scanner/tests/mocks/)
+
+file(COPY ${SRC_FOLDER}/wazuh_modules/vulnerability_scanner/schemas/scanVulnerabilityCandidate.fbs DESTINATION
```

Comment:
You can avoid the `COPY` operation using a `target_compile_definitions` which specifies compile definitions to use when compiling a given <target>. The named <target> must have been created by a command such as [add_executable()](https://cmake.org/cmake/help/latest/command/add_executable.html#command:add_executable) or [add_library()](https://cmake.org/cmake/help/latest/command/add_library.html#command:add_library) and must not be an [ALIAS target](https://cmake.org/cmake/help/latest/manual/cmake-buildsystem.7.html#alias-targets).

```cmake
target_compile_definitions(${PROJECT_NAME} PUBLIC SCHEMA_ROOT_PATH="${SRC_FOLDER}/wazuh_modules/vulnerability_scanner/schemas/")
```

And then in your `databaseFeddManager_test.hpp`

https://github.com/wazuh/wazuh/blob/460943386c69d6f6da80e790e82c3c5a299088c9/src/wazuh_modules/vulnerability_scanner/tests/databaseFeedManager/databaseFeedManager_tests.hpp#L35

Put 

```cpp
const std::string flatbufferSchema {SCHEMA_ROOT_PATH "scanVulnerabilityCandidate.fbs"};
```